client/systems.go

```diff
@@ -103,3 +103,34 @@ func (client *Client) DoSystemAction(systemLabel string, action *SystemAction) e
 	}
 	return nil
 }
+
+// DoSystemReboot issues a request to reboot into the given label/mode
```

Comment:
RebootToSystem issues a request to reboot into system with the given label and the given mode.

Reply:
Ups, silly me, thank you!